.orchestra/config/components.yml

```diff
@@ -17,6 +17,8 @@ environment:
   - ORCHESTRA_NODE_CACHE: $ORCHESTRA_ROOT/share/node_cache
   - HARD_FLAGS_YARN: "--non-interactive --no-progress --frozen-lockfile --offline"
   - TERMINFO: $ORCHESTRA_ROOT/share/terminfo
+  - WINEPREFIX: $ORCHESTRA_DOTDIR/wine-prefix
```

Comment:
What do you think of putting this in `.orchestra/cache`? So users can safely assume it's safe to drop it.

Reply:
We could do that, but isn't `.orchestra/cache` for orchestra-the-tool to store its data only?

Reply:
OK, let's not touch for now. But we need to concentrate all the disposable stuff in a single directory at a certain point (include `binary-archives`, `source-archives`, `tmproot`, `config_cache.json`, `config_cache.yml`, and `remote_refs_cache.json`).

But not now.